atmo/jobs/migrations/0012_add_job_run_history.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.11 on 2017-02-10 21:54
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def migrate_job_runs(apps, schema_editor):
+    SparkJob = apps.get_model('jobs', 'SparkJob')
+    SparkJobRun = apps.get_model('jobs', 'SparkJobRun')
+
+    for job in SparkJob.objects.all():
+        if job.current_run_jobflow_id:
+            SparkJobRun.objects.create(
+                spark_job=job,
+                jobflow_id=job.current_run_jobflow_id,
+                status=job.most_recent_status,
+                scheduled_date=job.last_run_date,
+            )
+
+
+def rollback_job_runs(apps, schema_editor):
+    SparkJob = apps.get_model('jobs', 'SparkJob')
+    SparkJobRun = apps.get_model('jobs', 'SparkJobRun')
+
+    for job in SparkJob.objects.all():
+        last_run = SparkJobRun.objects.filter(spark_job=job).latest()
+        if last_run:
```

Comment:
I think this may raise a `DoesNotExist` exception if there is for some reason no latest job run for the job, e.g. because the current_run_jobflow_id was not set and in the forward migration no job run was created.